docs/ios/troubleshooting/mtouch-errors.md

```diff
@@ -1983,7 +1983,7 @@ This error indicates a bug in Xamarin.iOS. Please file a new issue on [github](h
 
 <a name="MT4140" />
 
-### MT4140: The registrar found a signature mismatch in the method '*.*' - the selector indicates the variadic method takes * parameters, while the managed method has * parameters.
+### MT4140: The registrar found a signature mismatch in the method '\*.\*' - the selector indicates the variadic method takes \* parameters, while the managed method has \* parameters.
```

Comment:
Not sure if this was rendering incorrectly or was supposed to be a bolded period

Reply:
ditto